.drone.yml

```diff
@@ -107,6 +107,28 @@ pipeline:
       event: [push, tag]
       branch: [dev]
 
+  compile-gource-video:
```

Comment:
Could you put this in a group (along with `build-artefacts`) so that is builds in parallel?